src/Stratis.Bitcoin.Features.Consensus.Tests/Rules/CommonRules/BlockHeaderPosContextualRuleTest.cs

```diff
@@ -124,10 +127,10 @@ public BlockHeaderPosContextualRuleTest()
                 },
             };
             this.network.Consensus.LastPOWBlock = 12500;
-            this.ruleContext.BlockValidationContext.Block.Transactions[1].Time = this.ruleContext.BlockValidationContext.Block.Transactions[0].Time + 16;
-            this.ruleContext.BlockValidationContext.ChainedBlock.Header.Time = this.ruleContext.BlockValidationContext.Block.Transactions[0].Time + 15;
+            this.ruleContext.BlockValidationContext.Block.Transactions[1].Time = this.ruleContext.BlockValidationContext.Block.Transactions[0].Time + MaxFutureDriftAfterHardFork + 1;
+            this.ruleContext.BlockValidationContext.ChainedBlock.Header.Time = this.ruleContext.BlockValidationContext.Block.Transactions[0].Time + MaxFutureDriftAfterHardFork;
```

Comment:
here is where I meant

Reply:
Yes I think you are right here. It can be anything mismatching. Doesn't have anything to do with the drift value yet.